app/controllers/course/assessment/question/text_responses_controller.rb

```diff
@@ -45,8 +50,13 @@ def destroy
   def text_response_question_params
     params.require(:question_text_response).permit(
       :title, :description, :staff_only_comments, :maximum_grade, :weight, :allow_attachment,
+      :hide_text,
       skill_ids: [],
       solutions_attributes: [:_destroy, :id, :solution_type, :solution, :grade, :explanation]
     )
   end
+
+  def question_type
```

Comment:
I think you misunderstood this from `#display_title`, that is used for the `assessment#show`. `#display_title` shows things like `Question 3: Finding the number of buses` rather `"file upload question"` or `"text response question"`.

I decided against having `title` in the method name as it might be confused with `display_title` (I myself took a while to figure out which was what).